app/validators/proxy_features_validator.rb

```diff
@@ -0,0 +1,14 @@
+class ProxyFeaturesValidator < ActiveModel::EachValidator
+  def initialize(args)
+    @features = args
+    @features.delete(:class)
+    super
+  end
+
+  def validate_each(record, attribute, value)
+    if value && !value.has_feature?(@features[attribute])
+      msg = "does not have #{@features[attribute]} feature"
+      record.errors["#{attribute}_id"] << _(msg)
```

Comment:
`_(msg)` will translate, but the string hasn't been extracted.  The string must be within the _() to be extracted.